pages/linux/fdisk.md

```diff
@@ -10,3 +10,27 @@
 - Start the partition manipulator:
 
 `fdisk {{/dev/sdX}}`
+
+- Once partitioning a disk, create a partition:
+
+`n`
+
```

Comment:
This needs to be looked at, the syntax for adding memory disk allocation is `+{{size}}{{K|M|G|T|P}}`

To a beginner and me, that is odd. I would put 500M or 500MB or 500 MB, the be  confused that it didn't work and have to look it up on my phone, which is annoying enough (when installing arch, unable to use a web browser)

Reply:
The only possible solution I can see is maybe putting that in the description

Reply:
`fdisk` is an interactive command, so you don't specify the syntax for the size of the partition here - I'm confused what you're referring to here?

Reply:
At the command prompt, it is done during the command, 

EG

$ sudo fdisk /dev/sda 

new 

_Enter size of partition 1_ +500M

Reply:
You need to put a +, which IMO is an odd syntax